packages/components/tooltip/src/tooltip.spec.ts

```diff
@@ -53,6 +54,24 @@ describe('sl-tooltip', () => {
       expect(tooltip.matches(':popover-open')).to.be.false;
     });
 
+    it('should toggle the tooltip on focusin and Escape key pressed', async () => {
+      const focusinEvent = new Event('focusin', { bubbles: true });
+      button?.dispatchEvent(focusinEvent);
+      expect(tooltip.matches(':popover-open')).to.be.true;
+
+      await sendKeys({ press: 'Escape' });
+      expect(tooltip.matches(':popover-open')).to.be.false;
+    });
+
+    it('should toggle the tooltip on pointerover and Escape key pressed', async () => {
+      const focusinEvent = new Event('pointerover', { bubbles: true });
+      button?.dispatchEvent(focusinEvent);
```

Comment:
Same question here

Reply:
But here I want to check if the tooltip is closing by `Esc` when the tooltip was opened by pointerover, by hover.